packages/types/js.ts

```diff
@@ -21,10 +29,11 @@ export const ZJSWebsiteStateDisplay = z.object({
 export type TJSWebsiteStateDisplay = z.infer<typeof ZJSWebsiteStateDisplay>;
 
 export const ZJsAppStateSync = z.object({
-  person: ZPersonClient.nullish(),
+  person: ZJsPerson.nullish(),
```

Comment:
I think we don't need the `personId` (person.id) in the app state. Can we just store the `userId` here instead?